src/DIPS.Xamarin.UI/Controls/RadioButtonGroup/RadioButtonGroup.xaml.cs

```diff
@@ -0,0 +1,323 @@
+            }
+
+            radioButtonGroup.Initialize(newItemsSource);
+        }
+
+        private void CollectionChanged(object sender, NotifyCollectionChangedEventArgs e)
+        {
+            Initialize(e.NewItems);
+        }
+
+        private static void OnSelectedColorPropertyChanged(BindableObject bindable, object oldvalue, object newvalue)
+        {
+            if (!(bindable is RadioButtonGroup radioButtonGroup)) return;
+            if (!(newvalue is Color newColor)) return;
+
+            radioButtonGroup.m_radioButtons.ForEach(
+                rb =>
+                {
+                    rb.SelectedColor = newColor;
+                    rb.RefreshColor(rb.IsSelected);
+                });
+        }
+
+        private static void OnDeSelectedColorPropertyChanged(BindableObject bindable, object oldvalue, object newvalue)
+        {
+            if (!(bindable is RadioButtonGroup radioButtonGroup))
+                return;
+            if (!(newvalue is Color newColor))
+                return;
+
+            radioButtonGroup.m_radioButtons.ForEach(
+                rb =>
+                {
+                    rb.DeSelectedColor = newColor;
+                    rb.RefreshColor(rb.IsSelected);
+                });
+        }
+
+        /// <summary>
+        ///     Get the value from the <see cref="DisplayMemberPath" />
+        /// </summary>
+        /// <param name="item">The object to look for the value of the property</param>
+        /// <returns></returns>
+        private string GetDisplayPath(object item)
+        {
+            if (string.IsNullOrEmpty(DisplayMemberPath))
+            {
+                return item == null ? string.Empty : item.ToString();
+            }
+
+            if (m_displayMember == null)
+            {
+                m_displayMember = item.GetType().GetProperty(DisplayMemberPath);
+            }
+
+            var value = m_displayMember?.GetValue(item, null);
+            return value == null ? string.Empty : value.ToString();
+        }
+
+        private void Initialize(IEnumerable newItems)
+        {
+            if (newItems == null) return;
+
+            var row = radioButtonContainer.Children.Count;
+
+            //Add separator before the first element
+            if (row == 0)
+            {
+                AddSeparator(row);
+                row++;
+            }
+
+            foreach (var item in newItems)
+            {
+                var radioButton = new RadioButton() { Text = GetDisplayPath(item), Identifier = item };
+                //Initialize radio button
+                radioButton.Initialize(this);
+
+                //Set colors for each radiobutton
+                radioButton.SelectedColor = SelectedColor;
+                radioButton.DeSelectedColor = DeSelectedColor;
+                radioButton.RefreshColor(radioButton.IsSelected);
+
+                //Set margin for each button
+                radioButton.Padding = new Thickness(0, 15, 0, 15);

```

Comment:
15 = resource?

Reply:
Code-Comment is also wrong, its padding not margin